app/code/Magento/Sales/Model/Order/Email/SenderBuilder.php

```diff
@@ -28,29 +26,19 @@ class SenderBuilder
      */
     protected $transportBuilder;
 
-    /**
-     * @var TransportBuilderByStore
-     */
-    private $transportBuilderByStore;
-
     /**
      * @param Template $templateContainer
      * @param IdentityInterface $identityContainer
      * @param TransportBuilder $transportBuilder
-     * @param TransportBuilderByStore $transportBuilderByStore
      */
     public function __construct(
         Template $templateContainer,
         IdentityInterface $identityContainer,
-        TransportBuilder $transportBuilder,
-        TransportBuilderByStore $transportBuilderByStore = null
```

Comment:
As part of 2.2 release line this change is not backward compatible. It's better to keep it in constructor and not to use inside class.
As part of 2.3 release line while it's still not released - we can do such change

Reply:
Done